usaon_vta_survey/__init__.py

```diff
@@ -30,6 +30,7 @@
 db.init_app(app)
 
 app.jinja_env.globals.update(sqla_inspect=sqla_inspect)
+app.jinja_env.globals.update(__version__=__version__)
```

Comment:
We don't need two calls here, we can just pass both in as separate keyword arguments to the first call.

Reply:
done :)